src/Maps/MapWriter.cpp

```diff
@@ -89,7 +89,7 @@ namespace MapWriter {
 		{
 			WriteContainerSize(streamWriter, tileGroups.size());
 
-			streamWriter.SeekRelative(sizeof(int));
+			streamWriter.SeekRelative(sizeof(int32_t));
```

Comment:
I assume you mean SeekableStreamWriter. I prefer using the SeekRelative here because it is easier to read and understand what is happening then introducing the noise of a dummy variable.

However, this is the only part of the code using a feature that requires SeekableStreamWriter, so switching to a dummy variable would allow relaxing the entire Write process to accepting a StreamWriter. But, we don't have a defined StreamWriter to take advantage of the situation or a planned use case for such an advantage anyways.

Reply:
On retrospect, let us go ahead and use a temp value here.